src/test/java/net/sf/jabref/logic/importer/fileformat/BibTeXMLImporterTest.java

```diff
@@ -34,11 +34,12 @@
      * @throws IOException
      */
     public List<Path> getTestFiles() throws IOException {
-        List<Path> files = new ArrayList<>();
-        try (DirectoryStream<Path> stream = Files.newDirectoryStream(Paths.get(FILEFORMAT_PATH))) {
-            stream.forEach(files::add);
+        try (Stream<Path> stream = Files.list(Paths.get(BibTeXMLImporterTest.class.getResource("").toURI()))) {
+            return stream.filter(p -> !Files.isDirectory(p)).collect(Collectors.toList());
+        } catch (URISyntaxException e) {
```

Comment:
Would it make sense to just throw this Exception from the method instead? I guess the test will/should fail anyway.